diff --git a/src/EventSubscriber/ConfigDevelAutoExportSubscriber.php b/src/EventSubscriber/ConfigDevelAutoExportSubscriber.php
index 8c6f51d..ea25811 100644
--- a/src/EventSubscriber/ConfigDevelAutoExportSubscriber.php
+++ b/src/EventSubscriber/ConfigDevelAutoExportSubscriber.php
@@ -86,7 +86,9 @@ class ConfigDevelAutoExportSubscriber extends ConfigDevelSubscriberBase implemen
    */
   protected function autoExportConfig(Config $config) {
     $config_name = $config->getName();
-    $file_names = array_filter($this->getSettings()->get('auto_export'), function ($file_name) use ($config_name) {
+    $auto_export_array = $this->getSettings()->get('auto_export');
+    $auto_export_array = isset($auto_export_array) ? $auto_export_array : [];
+    $file_names = array_filter($auto_export_array, function ($file_name) use ($config_name) {
       return basename($file_name, '.' . FileStorage::getFileExtension()) == $config_name;
     });
     $this->writeBackConfig($config, $file_names);
diff --git a/src/EventSubscriber/ConfigDevelAutoImportSubscriber.php b/src/EventSubscriber/ConfigDevelAutoImportSubscriber.php
index 54d0993..3d53b12 100644
--- a/src/EventSubscriber/ConfigDevelAutoImportSubscriber.php
+++ b/src/EventSubscriber/ConfigDevelAutoImportSubscriber.php
@@ -20,7 +20,9 @@ class ConfigDevelAutoImportSubscriber extends ConfigDevelSubscriberBase implemen
   public function autoImportConfig() {
     $config = $this->getSettings();
     $changed = FALSE;
-    foreach ($config->get('auto_import') as $key => $file) {
+    $configurations = $config->get('auto_import');
+    $configurations = isset($configurations) ? $configurations : [];
+    foreach ($configurations as $key => $file) {
       if ($new_hash = $this->importOne($file['filename'], $file['hash'])) {
         $config->set("auto_import.$key.hash", $new_hash);
         $changed = TRUE;
